Cookbook/Technical-Documents/FastlaneMatch.md

```diff
@@ -1,65 +1,74 @@
-
-To install development certificates locally, run:
+1. Add shared developer account credentials from 1Password to your Xcode settings in `Preferences -> Accounts` menu. 
+2. Run 
 
 ```shell
- $ bundle exec fastlane match development --team_id <enterprise team id>
+ $ bundle exec fastlane match {{Type}} --team_id {{Team-ID}} 
 ```
 
-On the first run you will be asked a password to decrypt the repository managed by Match. The password is stored in the teams 1Password vault. After you entered the password Match will install latest development certificates for all the app targets listed in the `fastlane/Matchfile` in `app_identifier` function (not scoped to particular lane with `for_lane` function). 
-Match checks if the certificates and profiles are valid and will prompt you if there is an issue. By default it is configured to run in readonly mode so it will not regenerate any certificates or profiles if they are not valid.
-
-If you need to build the app locally and run it on a device that is not yet added to the development portal you need to follow these steps:
+Note:
+* We have 2 developer teams - Enterprise and App-store 
+* Following options can be used for the command above:
 
-- register the device manually on the portal. Make sure you do that in the correct team: it should be added to the enterprise team, not to the AppStore team
-- run Match to update development certificates and include this new device:
-
-```
-bundle exec fastlane match development --team_id <enterprise team id> --force_for_new_devices --readonly false
-```
+| Code signing identities for | Type param| Team-ID param    |
+|-----------------------------|-----------|-------------------
+|Day-to-day development       |development|Enterprise team ID|
+|Hockeyapp                    |enterprise |Enterprise team ID|
 
-This will regenerate the development profiles and certificates, including all new registered devices in them.
+3. On prompt for password to decrypt the repository, enter the one from 1Password vault.
 
-Certificates and provisioning profiles for enterprise builds (Hockey App) or release builds (Testflight and the App Store) can also be downloaded. Normally, this is not needed – as those tasks are usually executed on Circle CI and not locally – but might be necessary for some tasks.
+To troubleshoot, refer to `fastlane/Matchfile`, `fastlane/Appfile` and [documentation](https://docs.fastlane.tools/actions/match/)
 
 ### TestFlight & AppStore
 
 To install certificates and provisioning profiles for creating a Testflight or App Store build locally, run the command
 
 ```shell
- $ bundle exec fastlane match appstore --git_branch <appstore team id> --team_id <appstore team id> --app_identifier <appstore bundle id>
+ $ bundle exec fastlane match appstore --team_id <appstore team id> --git_branch <appstore team id> --app_identifier <appstore bundle id>
```

Comment:
I'd rather use the same placeholder names and syntax throughout the whole document. So replacing `<appstore team id>` with `((Team-ID}}` (nit: I even prefer `{{TeamID}}` without the dash), and use always the same delimiters (either curly braces or angle brackets but not a mix of both)

Reply:
Good observation. I had left the previous commands pretty much untouched and for the ones I wrote, I used curly braces